tests/parametric/test_sampling_span_tags.py

```diff
@@ -0,0 +1,141 @@
+from utils.parametric.spec.trace import MANUAL_DROP_KEY  # noqa
+from utils.parametric.spec.trace import MANUAL_KEEP_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_AGENT_PRIORITY_RATE  # noqa
+from utils.parametric.spec.trace import SAMPLING_DECISION_MAKER_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_PRIORITY_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_RULE_PRIORITY_RATE  # noqa
+from utils.parametric.spec.trace import Span  # noqa
+from utils.parametric.spec.trace import find_span_in_traces  # noqa
+
+
+def _get_spans(test_agent, test_library):
+    with test_library:
+        with test_library.start_span(name="parent", service="webserver") as ps:
+            with test_library.start_span(name="child", service="webserver", parent_id=ps.span_id):
+                pass
+
+    traces = test_agent.wait_for_num_spans(2, clear=True)
+
+    parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+    child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+    return parent_span, child_span, traces[0][0]
+
+
+def _assert_sampling_tags(parent_span, child_span, first_span, dm, parent_priority, parent_rate):
+    if dm is not None or "meta" in first_span:
+        assert first_span["meta"].get(SAMPLING_DECISION_MAKER_KEY) == dm
+    assert parent_span["metrics"].get(SAMPLING_PRIORITY_KEY) == parent_priority
+    assert parent_span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == parent_rate
+    assert child_span.get("metrics", {}).get(SAMPLING_RULE_PRIORITY_RATE) is None
+    if child_span != first_span:
+        assert child_span.get("meta", {}).get(SAMPLING_DECISION_MAKER_KEY) is None
+
+
+@scenarios.parametric
+class Test_Sampling_Span_Tags:
+    @pytest.mark.parametrize("library_env", [{"DD_TRACE_SAMPLE_RATE": 1}])
+    @bug(library="python", reason="Python sets dm tag on child span")
+    @bug(library="python_http", reason="Python sets dm tag on child span")
+    @bug(library="nodejs", reason="NodeJS does not set priority on parent span")
+    @bug(library="ruby", reason="ruby does not set dm tag on first span")
+    @bug(library="dotnet", reason="dotnet does not set dm tag on first span")
+    def test_tags_child_dropped_sst001(self, test_agent, test_library):
+        with test_library:
+            with test_library.start_span(name="parent", service="webserver") as parent_span:
+                with test_library.start_span(
+                    name="child", service="webserver", parent_id=parent_span.span_id
+                ) as child_span:
+                    child_span.set_meta(MANUAL_DROP_KEY, None)
+
+        traces = test_agent.wait_for_num_spans(2, clear=True)
+
+        parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+        child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+
+        _assert_sampling_tags(parent_span, child_span, traces[0][0], "-3", 2, 1)
```

Comment:
This would make more sense to me:

```suggestion
        _assert_sampling_tags(parent_span, child_span, traces[0][0], "-3", -1, 1)
```